src/accordion.test.visuals.ts

```diff
@@ -1,98 +1,85 @@
 import { expect, test } from '@playwright/test';
+import type GlideCoreAccordion from './accordion.js';
 
-test.describe('Accordion', () => {
-  test.describe('Light Mode', () => {
-    test('open', async ({ page }, test) => {
-      await page.goto('/iframe.html?id=accordion--accordion');
-
-      const handle = await page.waitForSelector('glide-core-accordion');
-
-      await handle?.evaluate((node) => {
-        node.open = true;
-        return node.updateComplete;
-      });
-
+test.describe('accordion--accordion', () => {
```

Comment:
Using the story ID here makes it clearer that what's being described is a story. "Accordion", as it was before, could be anything. And it wasn't clear that "Accordion" referred a story until you got to `"describe('With Icons')"`.

Another benefit is that the `page.goto` (just below) can now be based off `test.titlePath` instead of being hardcoded. That makes copy-pasting within and among test suites very easy and means one less thing to change if a story's name is changed.